### ISSUE TITLE
Publish on PyPI

### ISSUE THREAD

**Issue by jtpio** [r-wasm/jupyterlite-webr-kernel#1]
Really cool project, thanks for setting that up!

I was wondering if you were planning to distribute the kernel on PyPI at some point?

There are some on-going discussions about adding this kernel to the Try Jupyter demo site: https://github.com/jupyter/try-jupyter/issues/30

Having a published package on PyPI would make it easier to include in other JupyterLite deployments.

**Comment by jtpio** [comment]
For such projects we usually use the Jupyter Releaser to help make releases: https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_release_from_repo.html

For example the Xeus Python uses the releaser in case you would like to have a look: https://github.com/jupyterlite/xeus-python-kernel

I'm also happy to help set it up if needed.

**Comment by stevejpurves** [comment]
+1 for this as using the Jupyter Releaser would also mean the source extension package becomes available on npm.

**Comment by ericvd-ucb** [comment]
+1 for this project!  I could see a huge demand for this and have big deployment that can use this

**Comment by joelostblom** [comment]
I'm currently looking for a project that can support both Python and R interactively in the browser. JupyterLite seems like an ideal candidate if this WebR kernel becomes available on PyPI. I saw that there was recently a commit to make the package compatible with pip in https://github.com/r-wasm/jupyterlite-webr-kernel/commit/0d67ab3d0fd787b588fa1e24408e4e499124aa03, is the plan to also upload to PyPI?

**Comment by georgestagg** [comment]
Yes, the plan is eventually to upload to PyPI. However, I'd like a few features working before initial release there:

* A more stable webR API, so that the specific version of R/webR can be swapped out by the Jupyterlite user.
* A better way for the webR filesystem to interact with the existing jupyterlite file system.

**Comment by georgestagg** [comment]
Now available at https://pypi.org/project/jupyterlite-webr/
